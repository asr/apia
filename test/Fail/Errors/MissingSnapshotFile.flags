--atp=e --snapshot-test --snapshot-dir=no-dir
