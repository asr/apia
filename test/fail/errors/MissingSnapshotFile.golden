ret > ExitFailure 1
err > apia: the file no-dir/test/fail/errors//MissingSnapshotFile/16-foo.fof does not exist
err >
